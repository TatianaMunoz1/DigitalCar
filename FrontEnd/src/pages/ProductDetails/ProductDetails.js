import React from "react";
import styles from "./ProductDetails.module.css";
import SectionProductDetail from "../../components/SectionProductDetail/SectionProductDetail";

function ProductDetails() {
  return (
    <div className={styles.conteiner}>
      <SectionProductDetail />
    </div>
  );
}

export default ProductDetails;
