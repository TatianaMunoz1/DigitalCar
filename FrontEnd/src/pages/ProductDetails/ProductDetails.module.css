* {
  box-sizing: border-box;
  margin: 0%;
}
.conteiner {
  background-color: #000;
}
