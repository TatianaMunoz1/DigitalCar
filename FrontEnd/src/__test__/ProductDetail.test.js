import React from "react";
import { mount, shallow } from "enzyme";
import "@testing-library/jest-dom/extend-expect";
import { render, screen } from "@testing-library/react";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";
import ProductDetails from "../pages/ProductDetails/ProductDetails";


describe(" Product Details", () => {

    test("render  Product Details", () => {
        const wrapper = shallow(<ProductDetails/>)
        expect(wrapper).toMatchSnapshot();
    });
});