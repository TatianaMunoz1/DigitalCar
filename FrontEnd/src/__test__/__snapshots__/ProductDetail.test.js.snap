// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Product Details render  Product Details 1`] = `ShallowWrapper {}`;
